src/io.c

```diff
@@ -122,7 +122,7 @@ static void avifIOFileReaderDestroy(struct avifIO * io)
     avifFree(io);
 }
 
-avifIO * avifIOCreateFilePtrReader(FILE * f, avifBool closeOnDestroy)
+static avifIO * avifIOCreateFilePtrReader(FILE * f, avifBool closeOnDestroy)
```

Comment:
As an internal function, we don't need the closeOnDestroy parameter, and we don't need the reader->closeOnDestroy field, because they are always set to true.

Reply:
Done.